packages/ibm-products-web-components/src/components/interstitial-screen/interstitial-screen-header.ts

```diff
@@ -92,6 +93,17 @@ class CDSInterstitialScreenHeader extends SignalWatcher(
     );
   }
 
+  getElementForAriaLive = () => {
+    const currentStep = interstitialDetailsSignal.get().currentStep;
+    const stepDetails = interstitialDetailsSignal.get().stepDetails;
+    return html` <div
+      aria-live="polite"
+      aria-atomic="true"
+      class="${carbonPrefix}--visually-hidden"
+    >
+      Step ${currentStep + 1} of ${stepDetails.length}
```

Comment:
We might want to think about if there's a better way for globalization support here